### PR TITLE
Set up Typescript for pyrene-graphs and tuktuktwo

### PR DESCRIPTION
**Pull request by sosostris**
This PR sets up Typescript in the same way as in Pyrene, which is prerequisite to using Storybook for the visualization components.

Based on the last discussion with designer, we're going to have a "Visualizations" section parallel to the "Components" section in Storybook, where all visiaulization-related components (all components in tuktuktwo and pyrene-graphs) will be displayed, and no nested structure.

Since the current kitchensink is not displayed tuktuktwo components anyway, the hack for the `docgen` and `typescript-to-proptypes` hack is not added.

### CODE REVIEW

File: pyrene-graphs/jest.config.js [hunk 1]
```diff
@@ -168,6 +168,8 @@ module.exports = {
   transform: {
     '^.+\\.js$': 'babel-jest',
     '^.+\\.jsx?$': 'babel-jest',
+    '^.+\\.ts$': 'babel-jest',
+    '^.+\\.tsx?$': 'babel-jest',
```
Comment (cyrilwanner) [inline pyrene-graphs/jest.config.js:172]: I think the second line already covers the first one (`x?` -> x is optional and it matches both `.ts` and `.tsx`). Same for the js part.

File: tuktuktwo/jest.config.js [hunk 2]
```diff
@@ -168,6 +168,8 @@ module.exports = {
   transform: {
     '^.+\\.js$': 'babel-jest',
     '^.+\\.jsx?$': 'babel-jest',
+    '^.+\\.ts$': 'babel-jest',
+    '^.+\\.tsx?$': 'babel-jest',
```
Comment (cyrilwanner) [inline tuktuktwo/jest.config.js:172]: Same here

**Review by cyrilwanner — APPROVED** [review]
Thank you, it looks good to me!
Just one suggestion to simplify a config file, it you want to apply this :)

**Review by Sakai-san — APPROVED** [review]
Thanks a lot !!

File: pyrene-graphs/jest.config.js (hunk 1 above)
Comment (sosostris) [reply pyrene-graphs/jest.config.js:172]: I totally agree! lol ! Thanks for the sharp eye :) Let me clean this up!

**Pull request merged**